Scripts/models/logit.py

```diff
@@ -406,20 +403,19 @@ def _calc_utils(self, impedance):
             logsum = pandas.Series(numpy.log(expsum), self.purpose.zone_numbers)
             label = self.purpose.name + "_" + mode[0]
             self.zone_data._values[label] = logsum
-            self.resultdata.print_data(
-                logsum, "accessibility.txt",
-                self.zone_data.zone_numbers, label)
+            self.resultdata.print_data(logsum, "accessibility.txt", label)
         return self._calc_mode_util(self.dest_expsums)
 
     def _calc_prob(self, mode_expsum):
         prob = {}
         self.mode_prob = {}
-        self.dest_prob = {}
+        self.cumul_dest_prob = {}
         for mode in self.mode_choice_param:
             self.mode_prob[mode] = self.mode_exps[mode] / mode_expsum
             dest_expsum = self.dest_expsums[mode]["logsum"]
-            self.dest_prob[mode] = self.dest_exps[mode].T / dest_expsum
-            prob[mode] = self.mode_prob[mode] * self.dest_prob[mode]
+            dest_prob = self.dest_exps[mode].T / dest_expsum
+            prob[mode] = self.mode_prob[mode] * dest_prob
+            self.cumul_dest_prob[mode] = dest_prob.cumsum(axis=0)
```

Comment:
@zptro I can't figure out why axis is zero (sum over rows for each column). If matrix is based on impedance, should it be other way around `axis=1`?

Reply:
Yes, I agree this is a bit confusing. The reason is found on row 416, where the `self.dest_exps[mode]` matrix is transposed. An alternative to transposing the matrix would be to transform vectors `dest_expsum` and `self.mode_prob[mode]` (and later also `tours` vector in `purpose.calc_demand`) into column vectors using `numpy.newaxis`, but I thought this solution was a bit more elegant.